pkg/controller/versions/strategy_v1_1.go

```diff
@@ -215,21 +215,6 @@ func (v *versionStrategyV1_1) ValidateDowngrade(ctx context.Context, cl client.C
 		})
```

Comment:
Shouldn't whole this function be removed?

Reply:
I believe that's part of the interface, so it needs to be there.  It should return an error, as 1.0 is not supported (i.e. you can't downgrade).

Reply:
tbh what I never understood about these func is that they're called on downgrade *to* that version and also on downgrade *from* it. So when I'm downgrading 2.1 to 2.0, ValidateDowngrade is called for both, which is really confusing

Reply:
nevermind, I was mistaking a `<` for a `<=`. This is only called on the downgrade *to* a version.

Reply:
func is throwing an error now